MainWindow.xaml

```diff
@@ -1,17 +1,17 @@
-﻿<Window x:Class="Pikuniku_Save_Manager.MainWindow"
+<Window x:Class="Pikuniku_Save_Manager.MainWindow"
         xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
         xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
         xmlns:local="clr-namespace:Pikuniku_Save_Manager"
         mc:Ignorable="d"
-        Title="Pikuniku Save Manager" Height="220" Width="600" ResizeMode="NoResize">
+        Title="Pikuniku Save Manager" Height="220" Width="600" ResizeMode="CanResize">
```

Comment:
The reason I had to NoResize flag here was because the comboboxes would get stretched and look really bad if the window was resized.